### PR TITLE
chore(dev): install latest k3d

### PR DESCRIPTION
**Pull request by akwan**
If you want to fetch the latest `k3d` whenever building the homelab-tools docker image. Inspired from https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8

### CODE REVIEW

**Comment by khuedoan** [comment]
Should this just be `https://github.com/k3d-io/k3d/releases/latest/download/k3d-linux-amd64`?

**Comment by khuedoan** [comment]
Btw feel free to add yourself here :wink:

**Comment by akwan** [comment]
Well that's embarassing. 😆 Glad there's an easier way!

File: tools/Dockerfile [hunk 1]
```diff
@@ -41,9 +41,9 @@ RUN pacman --sync --refresh --noconfirm \
     terraform \
     yamllint
 
-# TODO better way to install k3d?
-RUN curl -L https://github.com/k3d-io/k3d/releases/download/v5.4.1/k3d-linux-amd64 > /usr/local/bin/k3d \
-    && chmod +x /usr/local/bin/k3d
+ARG K3D_PATH=/usr/local/bin/k3d
+RUN curl -L https://github.com/k3d-io/k3d/releases/latest/download/k3d-linux-amd64 -o $K3D_PATH -s \
```
Comment (khuedoan) [inline tools/Dockerfile:45]: I would prefer using the current `/usr/local/bin/k3d` instead of another `ARG` (unneeded layer)

**Review by khuedoan — APPROVED** [review]
LGTM, thanks!

**Pull request merged**